onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -1056,7 +1056,52 @@ CUDAExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph,
       result.push_back(std::make_unique<ComputeCapability>(std::move(sub_graph)));
     }
   }
+
+  if (enable_compiler_) {
+#ifdef USE_TENSORRT
```

Comment:
>USE_TENSORRT [](start = 7, length = 12)

when this is not defined, enable_compiler_ would do nothing silently. Please give some warning message at least, if not treating as error.